include/ck/tensor_operation/gpu/device/device_batchnorm_forward.hpp

```diff
@@ -20,18 +20,20 @@ struct DeviceBatchNormFwd : public BaseOperator
         const std::array<index_t, Rank> xyLengths,
         const std::array<index_t, Rank> xStrides,
         const std::array<index_t, Rank> yStrides,
+        const std::array<int, NumBatchNormReduceDim> reduceDims,
```

Comment:
use index_t

Reply:
Should not use `index_t` here,  `reduceDims` is not strides or lengths, it only provides information for which dimensions to reduce

Reply:
Fixed by [d4b1c](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/403/commits/eac10f802c5baf0c95b4f24be10955ae3b3d4b1c)